.gitignore

```diff
@@ -1,3 +1,7 @@
+# TSV and JSON files for testing / evaluation
+*.tsv
+*.json
```

Comment:
Try to limit this to only first level, otherwise future JSON files in fixtures will not be added and we will be scratching our heads for a long time. What I tend to do is to create a local `./tmp`, ignore it, and use that for all local and temporary files.

Reply:
yes that sounds nice, great idea